### ISSUE TITLE
getAttachmentContentAsync fails to get Attachments when waiting 5 - 10 minutes

### ISSUE THREAD

**Issue by eneoli** [OfficeDev/office-js#644]
## Expected Behavior
getAttachmentContentAsync should return the asked attachment.

## Current Behavior
It works perfectly, when you request the attachment immediately, but when waiting, there will be an error instead.

## Steps to Reproduce, or Live Example
 wait 5 - 10 minutes
Office.context.mailbox.item.getAttachmentContentAsync(id)

## Context
Trying to get attachments from mail and send them to our web app. (Read mail)

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: Office Online
* Host [Excel, Word, PowerPoint, etc.]: Outlook OWA
* Office version number: latest OWA
* Operating System: linux
* Browser (if using Office Online): Chrome

## Useful logs
GET https://outlook.live.com/owa/.... 440

Access to fetch at 'https://outlook.live.com/owa... (redirected from 'https://attachment.outlook.live.net/owa/..../service.svc/s/GetFileAttachment?... from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
tslib.es6.js:69 Uncaught (in promise) TypeError: Failed to fetch

**Comment by exextoc** [comment]
@EneOli  Thanks for reporting this issue. It has been put on our backlog. We, unfortunately, have no timelines to share at this point.
Can you share the use case due which you are waiting for 5-10 min after getting the attachment and then getting its content?

**Comment by eneoli** [comment]
The Add In is part of our ticket system. It might be possible, that the service employee gets a call from the customer or calls the customer while the mail is open.

**Comment by exextoc** [comment]
@EneOli Thanks for reporting this issue and sharing the usecase. We have already added this to our backlog.

**Comment by JuaneloJuanelo** [comment]
known issue, item in backlog, so closing issue

**Comment by GarryLowther** [comment]
This is not the only problem, my code uses getAttachmentContentAsync and it can get attachment data in Outlook 365 Web no problem however in the desktop installed Outlook, it returns 
 "HostError: the operation is not supported"

**Comment by exextoc** [comment]
@TriSysBusinessSoftware : We recommend opening another issue on github regarding what you are facing. This way we can track and have a quicker turn around time in responding back to posts. Could you open another issue on github explaining the issue you are facing and we'll look into it. Thanks.

**Comment by eneoli** [comment]
There is another problem:
If calling the function after 5-10 minutes it throws the following error: 

`Uncaught (in promise) o {message: "Invalid token specified", filename: "", lineno: 0, colno: 0}`

The problem is, that the error is thrown within an setInterval, so there is absolutely no chance to catch it....

![image](https://user-images.githubusercontent.com/23139864/115015428-e7c95c80-9eb3-11eb-9043-81092a759974.png)

**Comment by exextoc** [comment]
Apologies for delayed response. We tried this multiple times and not able to repro this now. Are you still able to repro this? Does it repro all the time, or it happens only few times?
